src/components/EventDetails/index.js

```diff
@@ -117,10 +117,12 @@ EventDetailsComponent.propTypes = {
     startDate: PropTypes.shape({
       nanoseconds: PropTypes.number.isRequired,
```

Comment:
I skimmed through this in the previous PR, just wondering why we need nanoseconds?

Reply:
Oh is that not how we do it for type validation? startDate comes in as an object with nanoseconds, seconds, and toDate so I thought that was what I had to do. Do we only need to check for type for something if we use them?

Reply:
you can use instanceOf :)

Reply:
Proptypes is starting to wear me out a lil, maybe we can see how people feel about using something like typescript during summer lol